wordpress-mobile/WordPress-Android
# Issue/6138 default image class

Fixes #6138.

To test:
1. Open Aztec editor
2. Insert a new image
3. Switch to HTML mode
4. Verify the <img> tag has a `size-full` class
